apps/pie-docs/src/assets/styles/components/_contentPage.scss

```diff
@@ -159,6 +159,10 @@
 
 .c-contentPage-header--hero + .c-contentPage-content {
     border-top: none;
+
+    .c-content-container {
```

Comment:
this isn't applying padding anywhere else other than the home page instance, is it?

Reply:
Nope, I've checked and Percy is happy with it too